openslides/motions/models.py

```diff
@@ -738,6 +738,9 @@ class MotionChangeRecommendation(RESTModelMixin, models.Model):
     rejected = models.BooleanField(default=False)
     """If true, this change recommendation has been rejected"""
 
+    internal = models.BooleanField(default=True)
```

Comment:
Default should be false so that all change recommendations can be seen by users that can see the motion (and discuss about the proposals etc.).